tests/pipeline/test_domain.py

```diff
@@ -76,7 +97,7 @@ def run(ts):
 
 class SpecializeTestCase(zf.ZiplineTestCase):
 
-    @parameter_space(domain=[US_EQUITIES, CA_EQUITIES, GB_EQUITIES])
+    @parameter_space(domain=DOMAINS)
```

Comment:
It's probably overkill to run these against every single domain we have. The purpose of these tests is to verify that `specialize` works with an arbitrary domain, but I don't think it's likely that this will pass on three domains but fail on a fourth. That said, these tests are super fast, so I'm not sure it matters.